packaging/opensuse/snapd.spec

```diff
@@ -14,17 +14,28 @@
 
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 
+# Test keys: used for internal testing in snapd.
 %bcond_with testkeys
+%if %{with testkeys}
+%global with_test_keys 1
+%endif
 
 # Enable AppArmor on openSUSE Tumbleweed (post 15.0) or higher
 # N.B.: Prior to openSUSE Tumbleweed in May 2018, the AppArmor userspace in SUSE
 # did not support what we needed to be able to turn on basic integration.
 %if 0%{?suse_version} >= 1550
-%bcond_without apparmor
-%else
-%bcond_with apparmor
+%global with_apparmor 1
```

Comment:
You're breaking bcond flags by doing this... Also, bconds already emit 1 or 0.

Cf. https://github.com/rpm-software-management/rpm/blob/rpm-4.14.x/macros.in#L101-L102

Reply:
Can you elaborate please? How should this look like?

Reply:
You should just use what it was before...